apps/client/src/common/components/utils/max-width-wrapper.tsx

```diff
@@ -11,7 +11,7 @@ const MaxWidthWrapper = ({
   return (
```

Comment:
### Critical Points:
❌  The change from `px-2.5` to `px-4` might affect the layout and spacing of the component. It's important to consider the impact of this change on the overall design and user experience.

### Good Points:
👍 The change to `px-4` might improve the visual appeal and readability of the content within the wrapper.